src/search/SearchSuggestions.js

```diff
@@ -28,16 +28,17 @@ export default function SearchSuggestions({ classes }) {
   const { state } = useContext(SearchContext)
 
   return (
-    <div className={classes.root}>
+    <>
       <LoadMask show={state.loading} transparent />
-
-      {state.groups &&
-        state.groups.map((group, i) => (
-          <div key={i} className={classes.group}>
-            <SearchSuggestionGroup {...group} />
-          </div>
-        ))}
-    </div>
+      <div className={classes.root}>
+        {state.groups &&
+          state.groups.map((group, i) => (
+            <div key={i} className={classes.group}>
```

Comment:
I know this is just reformatted code, but indexes really shouldn't be used as keys unless absolutely necessary. Seems like `group.caption` would work here, no?

Reply:
Changed